packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -814,8 +824,11 @@ class _PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
       position: position,
     )
     .then<void>((T newValue) {
-      if (!mounted || newValue == null)
+      if (!mounted || newValue == null) {
+        if (widget.onCancelled != null)
```

Comment:
if `!mounted`, you shouldn't call onCanceled.

Reply:
Ouch, that was kinda dumb.  Thanks for the catch.